GMX-Bots/Large-Open-and-Increase-Positions/README.md

```diff
@@ -0,0 +1,56 @@
+# Large Open/Increase Position Size Bot
+
+## Description
+
+This bot detects when an open/increase position is large. The bot listens to `UpdatePosition` and `IncreasePosition` event emissions on GMX's Vault Contract and creates a finding if the initial position `size` exceeds the `threshold` 
+
+> The `threshold` can be adjusted in **src/network.ts**.
+
+## Supported Chains
+
+- Arbitrum
+- Avalanche
+
+## Alerts
+
+- GMX-1-1
+  - Fired when the position `size` of the `UpdatePosition` event  which exceeds the `threshold` is equivalent to the `sizeDelta` of the `IncreasePosition` event
```

Comment:
First, this description is a bit messy. You can say that `sizeDelta` exceeds the `threshold` because that is the comparison you are doing, as in this case it is equal to size, the sentence is "correct", but that is not the ideal explanation. 
Second, from my understanding what you are trying to detect here is an opening which I can confirm reading the finding object description/name, so it will be worthy mentioning here that is fired when a newly Opened position is large.